depends/config.site.in

```diff
@@ -81,7 +81,7 @@ PKG_CONFIG="$(which pkg-config) --static"
 # These two need to remain exported because pkg-config does not see them
 # otherwise. That means they must be unexported at the end of configure.ac to
 # avoid ruining the cache. Sigh.
-export PKG_CONFIG_PATH="${depends_prefix}/share/pkgconfig:${depends_prefix}/lib/pkgconfig"
```

Comment:
In be1f13107d561f4a9601515aff1d74bdf650bbd7:
> share/pkgconfig directory is a wrong place to search for *.pc files.

Why is it the wrong place look? The `pkg-config` man page says `On most systems, pkg-config looks in /usr/lib/pkgconfig, /usr/share/pkgconfig, /usr/local/lib/pkgconfig and /usr/local/share/pkgconfig for these files.`

Reply:
> In [be1f131](https://github.com/bitcoin/bitcoin/commit/be1f13107d561f4a9601515aff1d74bdf650bbd7):
> 
> > share/pkgconfig directory is a wrong place to search for *.pc files.
> 
> Why is it the wrong place look? The `pkg-config` man page says `On most systems, pkg-config looks in /usr/lib/pkgconfig, /usr/share/pkgconfig, /usr/local/lib/pkgconfig and /usr/local/share/pkgconfig for these files.`

You're right. I erroneously relied on the online `pkg-config` [man page](https://linux.die.net/man/1/pkg-config) which missed the line you cited. And it is confirmed by the `pkg-config` source code I checked out.

Would "share/pkgconfig directory is a not place where dependency package *.pc files reside" a goog correction?